pyiceberg/catalog/glue.py

```diff
@@ -93,6 +93,13 @@
 if TYPE_CHECKING:
     import pyarrow as pa
 
+
+# There is a unique Glue metastore in each AWS account and each AWS region. By default, GlueCatalog chooses the Glue
+# metastore to use based on the user's default AWS client credential and region setup. You can specify the Glue catalog
+# ID through glue.id catalog property to point to a Glue catalog in a different AWS account. The Glue catalog ID is your
+# numeric AWS account ID.
+GLUE_ID = "glue.id"
```

Comment:
Shall we call it `GLUE_CATALOG_ID` since we use `glue_catalog_id` below?

Reply:
I went with `GLUE_ID` because it matches the Iceberg property (`glue.id`) and because the other property constants below this do the same.

I went with `glue_catalog_id` when it's used because it is (imo) a better name and maps more directly to the [AWS Glue terminology](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-database.html).